src/main/kotlin/com/sourcegraph/cody/autocomplete/action/AcceptAutocompleteActionHandler.kt

```diff
@@ -57,33 +20,17 @@ class AcceptAutocompleteActionHandler : AutocompleteActionHandler() {
     val project = editor.project ?: return
     val server = CodyAgent.getServer(project)
     if (server != null) {
-      val telemetry = CodyAutocompleteManager.instance.currentAutocompleteTelemetry
-      GraphQlLogger.logAutocompleteAcceptedEvent(project, telemetry?.params())
-      server.autocompleteClearLastCandidate()
-      acceptAgentAutocomplete(editor, maybeCaret)
-    } else {
       val caret = maybeCaret ?: getSingleCaret(editor) ?: return
-      AutocompleteText.atCaret(caret)?.let {
-        /* Log the event */
-        GraphQlLogger.logCodyEvent(project, "completion", "accepted")
-        WriteAction.run<RuntimeException> { applyAutocomplete(editor.document, it) }
```

Comment:
we remove `applyAutocomplete` here, but this is just fine since IIUC we `applyInsertText` now (which does the the same thing - applies the autocompletion)

Reply:
Correct, this case only exists in the scenario that the server shutsdown between the completion being returned and it being accepted. But we dont need the server to accept it except for telemetry purposes, so the case is imo exceptional enough that we dont need the extra code complexity of both cases